src/containers/Login/Login.js

```diff
@@ -1,6 +1,7 @@
 import React, { Component } from 'react';
 import { Link } from 'react-router-dom';
 import API from '../../utils/api';
+import InputFields from '../../components/InputFields/InputFields'
```

Comment:
Update with import of function for buildInputFields

Reply:
This should also now be changed...